### PR TITLE
Feat/climber profile

### PR DESCRIPTION
**Pull request by PulexIrritans**
Hi,
I have a added a climber profile. This sits in a new page which is linked to the navigation.
The profile itself is basically a form component where the details can be added and updated.
The information is posted to the database and also prefilled if data for the current climber exists in the database.

The user profile is needed as a preparation to generate user specific climb stats, and to create rankings with multiple users in the future.
Thanks for your review.

### CODE REVIEW

File: client/src/components/DateInput.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Input = styled.input`
+  height: 3rem;
+  padding: 0.5rem;
+  background-color: white !important;
```
Comment (mbosselmann) [inline client/src/components/DateInput.jsx:40]: Why do you need `!important`here? Usually, if `!important` is used that indicates that it is not exactly known why the declaration alone is not working. If my assumption is true you should check why you need to set `!important` and think about a better solution.

File: client/src/components/UserProfile.jsx [hunk 2]
```diff
@@ -0,0 +1,187 @@
+import TextInput from './TextInput';
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+import profilePic from '../images/ProfilePic.jpg';
+import Button from './Button';
+import DateInput from './DateInput';
+import dayjs from 'dayjs';
+const URL = process.env.REACT_APP_URL;
```
Comment (mbosselmann) [inline client/src/components/UserProfile.jsx:8]: I would prefer a blank line between your `imports` and your `consts` for better visibility of the latter. :)

File: client/src/components/UserProfile.jsx [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+        setClimber(data);
+        setSuccess(true);
+      })
+      .catch(error => {
+        setErrorSave(true);
+      });
+  };
+
+  return (
+    <>
```
Comment (mbosselmann) [inline client/src/components/UserProfile.jsx:66]: I think you don't need the fragments. Please check :)

**Review by mbosselmann — APPROVED** [review]
You rock! 🪨 Just some nit-picking comments. :) Keep it up! 💪

File: client/src/components/DateInput.jsx (hunk 1 above)
Comment (PulexIrritans) [reply client/src/components/DateInput.jsx:40]: @mbosselmann Thank you for the hint. !important was obviously quick and dirty. :-) I found out now how to do it properly with webkit-calendar-picker-indicator.

File: client/src/components/UserProfile.jsx (hunk 2 above)
Comment (PulexIrritans) [reply client/src/components/UserProfile.jsx:8]: done :-)

File: client/src/components/UserProfile.jsx (hunk 3 above)
Comment (PulexIrritans) [reply client/src/components/UserProfile.jsx:66]: You're right!

**Pull request merged**